src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -460,7 +431,7 @@ public override async ValueTask<(TValue value, string etag)> GetStateAndETagAsyn
         public override async ValueTask<bool> TryDeleteStateAsync(
             string storeName,
             string key,
-            string etag = default,
```

Comment:
eTag should be optional

Reply:
It wasn't clear to me when the etag is needed, because the `TrySaveStateAsync` method does defined it as needed and the documentation for `TryDeleteStateAsync` says "Tries to delete the the state associated with the provided key using the etag from the Dapr state." Also the `DaprClient.TryDeleteStateAsync` method defines a default for the etag param.

Reply:
Yes, it should be required for TryDelete.